Update guzzle to 7.5

abaracedo opened the pull request:
We're trying to update to guzzle to v7.5 (we use it in our project) and it's not possible as this dependency requires guzzle v6.5.

Can you upgrade it to the latest version? I've tested it pointing the project to my fork and it worked.

abaracedo committed:
Update guzzle to 7.5

mistermoper commented on composer.json:
```diff
@@ -4,7 +4,7 @@
     "require": {
         "ext-simplexml": "*",
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^7.5"
```
```suggestion
        "guzzlehttp/guzzle": "^6.3 || ^7"
```

omarlopesino committed:
Preserve compatibility with guzzle 6

Also let developers use guzzle lower than 7.5 in case they can't upgrade guzzle version.

omarlopesino commented:
Thank you for the contribution, and confirming it works well on your side after the Guzzle upgrade. I've just changed the composer.json so that it still allows guzzle ^6.3. Merged and released at v0.1.2 version.
